sdk/features/arm-features/review/arm-features.api.md

```diff
@@ -0,0 +1,343 @@
+    nextLink?: string;
+    value?: FeatureResult[];
+}
+
+// @public
+export interface FeatureProperties {
+    state?: string;
+}
+
+// @public
+export interface FeatureResult {
+    id?: string;
+    name?: string;
+    properties?: FeatureProperties;
+    type?: string;
+}
+
+// @public
+export interface Features {
+    get(resourceProviderNamespace: string, featureName: string, options?: FeaturesGetOptionalParams): Promise<FeaturesGetResponse>;
+    list(resourceProviderNamespace: string, options?: FeaturesListOptionalParams): PagedAsyncIterableIterator<FeatureResult>;
+    listAll(options?: FeaturesListAllOptionalParams): PagedAsyncIterableIterator<FeatureResult>;
+    listAllNext(nextLink: string, options?: FeaturesListAllNextOptionalParams): PagedAsyncIterableIterator<FeatureResult>;
+    listNext(resourceProviderNamespace: string, nextLink: string, options?: FeaturesListNextOptionalParams): PagedAsyncIterableIterator<FeatureResult>;
+    register(resourceProviderNamespace: string, featureName: string, options?: FeaturesRegisterOptionalParams): Promise<FeaturesRegisterResponse>;
+    unregister(resourceProviderNamespace: string, featureName: string, options?: FeaturesUnregisterOptionalParams): Promise<FeaturesUnregisterResponse>;
+}
+
+// @public
+export interface FeaturesGetOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesGetResponse = FeatureResult;
+
+// @public
+export interface FeaturesListAllNextNextOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesListAllNextNextResponse = FeatureOperationsListResult;
+
+// @public
+export interface FeaturesListAllNextOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesListAllNextResponse = FeatureOperationsListResult;
+
+// @public
+export interface FeaturesListAllOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesListAllResponse = FeatureOperationsListResult;
+
+// @public
+export interface FeaturesListNextNextOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesListNextNextResponse = FeatureOperationsListResult;
+
+// @public
+export interface FeaturesListNextOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesListNextResponse = FeatureOperationsListResult;
+
+// @public
+export interface FeaturesListOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesListResponse = FeatureOperationsListResult;
+
+// @public
+export interface FeaturesRegisterOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesRegisterResponse = FeatureResult;
+
+// @public
+export interface FeaturesUnregisterOptionalParams extends coreClient.OperationOptions {
+}
+
+// @public
+export type FeaturesUnregisterResponse = FeatureResult;
+
+// @public
+export const enum KnownSubscriptionFeatureRegistrationApprovalType {
```

Comment:
Let's make sure all enums are just regular enums. Const enums do not work well in javascript: https://github.com/Azure/autorest.typescript/issues/1013

Reply:
I think it's an important issue. @sarangan12 could you please fix it ASAP. I think it blocks the release.

Reply:
Created PR https://github.com/Azure/autorest.typescript/pull/1097